### PR TITLE
Vaulted card installments implementation

### PR DESCRIPTION
**Pull request by chetanjk**
Initiating installments flow based on eligibility and button flag for vaulted flow.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,10 +51,11 @@
...
   },
   "dependencies": {
-    "@paypal/checkout-components": "^5.0.30",
-    "@paypal/common-components": "^1.0.10",
+    "@paypal/checkout-components": "github:chetanjk/paypal-checkout-components#vaulted-installments",
+    "@paypal/common-components": "github:chetanjk/paypal-common-components#vaulted-installments",
+    "@paypal/installments-components": "github:chetanjk/paypal-installments-components#master",
     "@paypal/sdk-client": "^4.0.123",
-    "@paypal/sdk-constants": "^1.0.57",
+    "@paypal/sdk-constants": "github:chetanjk/paypal-sdk-constants#vaulted-installments",
```
Comment (bluepnume) [inline package.json:58]: Please revert

File: server/service/fundingEligibility.js [hunk 2]
```diff
@@ -87,9 +87,10 @@ function buildFundingEligibilityQuery(basicFundingEligibility : FundingEligibili
 
     const getCardQuery = () => {
         return {
-            eligible: graphqlTypes.boolean,
-            branded:  graphqlTypes.boolean,
-            vendors:  getCardVendorsQuery()
+            eligible:     graphqlTypes.boolean,
+            branded:      graphqlTypes.boolean,
+            installments: graphqlTypes.boolean,
```
Comment (bluepnume) [inline server/service/fundingEligibility.js:92]: We also need to make this change in `clientsdknodeweb` repo please.

**Comment by bluepnume** [comment]
Looks good, just one minor change

File: server/service/fundingEligibility.js (hunk 2 above)
Comment (chetanjk) [reply server/service/fundingEligibility.js:93]: done

File: package.json (hunk 1 above)
Comment (chetanjk) [reply package.json]: done

File: src/button/props.js [hunk 3]
```diff
@@ -180,6 +183,7 @@ export function getProps({ facilitatorAccessToken } : {| facilitatorAccessToken
         getPrerenderDetails,
         getPageUrl,
         enableThreeDomainSecure,
+        enableVaultInstallments,
```
Comment (chetanjk) [inline src/button/props.js:186]: @bluepnume, when can we default it to true?

**Comment by bluepnume** [comment]
Can you rebase against master? Looks like some failing tests that got fixed on master branch.

File: src/button/props.js (hunk 3 above)
Comment (bluepnume) [reply src/button/props.js:186]: Totally up to you. We release smart payment buttons approximately once per week, so feel free to enable whenever you're comfortable everything works.

**Comment by chetanjk** [comment]
Rebased

**Pull request merged**